pkgs/tools/security/step-ca/default.nix

```diff
@@ -2,29 +2,43 @@
 , lib
 , fetchFromGitHub
 , buildGoModule
+, coreutils
 , pcsclite
 , PCSC
 , pkg-config
+, hsmSupport ? true
 }:
 
 buildGoModule rec {
   pname = "step-ca";
-  version = "0.15.6";
+  version = "0.15.7-unstable";
```

Comment:
We should probably wait for the next release before we merge. This should only take a few days.